scripts/configs/offline_config.yaml

```diff
@@ -0,0 +1,36 @@
+# PG4 experiments.
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: {}
+  plan-only:
+    NAME: "oracle"
+    FLAGS: {}
+    
+ENVS:
+  deliver:
+    NAME: "pddl_easy_delivery_procedural_tasks"
+    FLAGS:
+      explore_random_options:
+      max_initial_demos: 1
+      num_train_tasks: 10
+      num_test_tasks: 10
+      max_num_steps_interaction_request: 10
+  spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS:
+      explore_random_options:
+      max_initial_demos: 1
+      num_train_tasks: 10
+      num_test_tasks: 10
+      max_num_steps_interaction_request: 10
+
+ARGS: []
+FLAGS:
+  strips_learner: oracle
+  sampler_learner: oracle
+  num_train_tasks: 50
+BRANCH: master
+START_SEED: 456
+NUM_SEEDS: 10
```

Comment:
this is just a convention, but let's keep a single new line at the end of the file